docs/pages/try-out-teleport/linux-server.mdx

```diff
@@ -80,8 +80,14 @@ Teleport's certificate.
 
 ## Step 2/6. Run a simple web service
 
-Run the following commands to create a directory on your Linux host
-called `demo-app` and add a simple HTML file to serve to clients:
+This guide assumes a web-based application for Teleport to provide (and restrict)
+access to. You can use any service that listens for HTTP requests, or create a demo
+application.
```

Comment:
```suggestion
Run a web application for Teleport to manage access to. You can use any service that listens for HTTP requests, or create a demo application using the instructions below.
```
Suggested to frame this paragraph as an instruction to the reader instead of a statement about the guide. Also suggesting connecting these instructions to the Details box.